infra/pr_helper.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env python
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+"""Adds comments for PR to provide more information for approvers."""
+import base64
+import json
+import os
+import subprocess
+
+import requests
+import yaml
+
+OWNER = 'google'
+REPO = 'oss-fuzz'
+GITHUB_URL = 'https://github.com/'
+API_URL = 'https://api.github.com'
+BASE_URL = f'{API_URL}/repos/{OWNER}/{REPO}'
+BRANCH = 'master'
+CRITICALITY_SCORE_PATH = '/home/runner/go/bin/criticality_score'
```

Comment:
I reckon `os.path.join()` is normally preferred, but this should be OK given we always run this script on GitHub?

Reply:
i think it's OK as we only run on ubuntu machines. But please let me know if we have style requirements for path.

Reply:
Yep, I am interested to learn that, too.

Reply:
Usually yes we would prefer os.path.oin. In this case though, even if we do os.path.join it's rather hardcoded to the Ubuntu GitHub actions runner so it would not be adding much value.